config/initializers/form_builder.rb

```diff
@@ -45,6 +47,11 @@ def has_errors?
 end
 
 module GOVUKDesignSystemFormBuilder
+  class FormBuilder < ActionView::Helpers::FormBuilder
+    delegate :content_tag, :tag, :safe_join, :link_to, :capture, to: :@template
+
+    include Roda::FormBuilder::Builder
```

Comment:
Could you also create a custom form builder that inherits from `GOVUKDesignSystemFormBuilder::FormBuilder` and then specifying it on line 1 (https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/1059/files#diff-bf1117bdafe92b2011b9ed317d701a0973101a9893f47509641c2d47f656df47L1), rather than injecting custom code into their class?

DFE Digital have been doing a reorganising of their code lately, and an internal change to this class has the potential to bite us in the future.

Reply:
Yeah, good call. It actually has bitten us here already as it goes - https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/1066